### PR TITLE
Fixed react warnings.

### LINKED ISSUE #2765: Fix React warnings in src/app/containers/PageHandlers/withPageWrapper/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**

1. Go to simorgh in terminal
2. Run npm run test:unit
3. Observe React warnings:

```
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: The prop `product` is marked as required in `Brand`, but its value is `undefined`.
          in Brand (created by BrandContainer)
          in BrandContainer (created by HeaderContainer)
          in header (created by HeaderContainer)
          in HeaderContainer (created by PageWrapper)
          in PageWrapper (created by PageWrapperContainer)
          in PageWrapperContainer
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: The prop `product` is marked as required in `Brand`, but its value is `undefined`.
          in Brand (created by BrandContainer)
          in BrandContainer (created by HeaderContainer)
          in header (created by HeaderContainer)
          in HeaderContainer (created by PageWrapper)
          in PageWrapper (created by PageWrapperContainer)
          in PageWrapperContainer
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: The prop `svg` is marked as required in `Brand`, but its value is `undefined`.
          in Brand (created by BrandContainer)
          in BrandContainer (created by HeaderContainer)
          in header (created by HeaderContainer)
          in HeaderContainer (created by PageWrapper)
          in PageWrapper (created by PageWrapperContainer)
          in PageWrapperContainer
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: The prop `product` is marked as required in `StyledBrand`, but its value is `undefined`.
          in StyledBrand (created by Brand)
          in Brand (created by BrandContainer)
          in BrandContainer (created by HeaderContainer)
          in header (created by HeaderContainer)
          in HeaderContainer (created by PageWrapper)
          in PageWrapper (created by PageWrapperContainer)
          in PageWrapperContainer
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: The prop `product` is marked as required in `StyledBrand`, but its value is `undefined`.
          in StyledBrand (created by Brand)
          in Brand (created by BrandContainer)
          in BrandContainer (created by HeaderContainer)
          in header (created by HeaderContainer)
          in HeaderContainer (created by PageWrapper)
          in PageWrapper (created by PageWrapperContainer)
          in PageWrapperContainer
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: The prop `svg` is marked as required in `StyledBrand`, but its value is `undefined`.
          in StyledBrand (created by Brand)
          in Brand (created by BrandContainer)
          in BrandContainer (created by HeaderContainer)
          in header (created by HeaderContainer)
          in HeaderContainer (created by PageWrapper)
          in PageWrapper (created by PageWrapperContainer)
          in PageWrapperContainer

```

**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #2765 

**Overall change:** 
Fixed react warnings.

**Code changes:**
- Wrapped ```PageWrapperHOC{}``` with ```ServiceContextProvider```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/PageHandlers/withPageWrapper/index.test.jsx [hunk 1]
```diff
@@ -13,6 +14,8 @@ describe('with pageWrapper', () => {
   const PageWrapperHOC = WithPageWrapper(PageWrapperContainer);
   shouldMatchSnapshot(
     `should render correctly`,
-    <PageWrapperHOC {...dataProps} />,
+    <ServiceContextProvider service="news">
```
Comment (chris-hinds) [inline src/app/containers/PageHandlers/withPageWrapper/index.test.jsx:17]: I don't think we need to be wrapping the PageWrapper with the service context. The PageWrapper receives the service name via a prop on the data object.

Whats the error that this was solving?

**Review by thekp — APPROVED** [review]

File: src/app/containers/PageHandlers/withPageWrapper/index.test.jsx (hunk 1 above)
Comment (thekp) [reply src/app/containers/PageHandlers/withPageWrapper/index.test.jsx:17]: i ran it locally, and it seems to fix these warnings:
(latest)
image
(branch)
image

Comment (chris-hinds) [reply src/app/containers/PageHandlers/withPageWrapper/index.test.jsx:17]: Ok this is fine then. In the app the PageWrapper is a child of the withContexts provider so this makes sense.

**Review by chris-hinds — APPROVED** [review]

**Comment by thekp** [comment]
this is just updated unit tests, doesn't need manual testing

**Pull request merged**